<launch>
  <node pkg="evdev_joy" type="evdev_joystick_node" name="joystick" output="screen">
        <param name="device_file_path" value="/dev/input/event19"/>
        <param name="max_send_frequency" value="30"/>
        <rosparam param="buttons_mapping">  [BTN_NORTH,
                                            BTN_EAST,
                                            BTN_SOUTH,
                                            BTN_WEST,
                                            BTN_TL,
                                            BTN_TR,
                                            BTN_SELECT,
                                            BTN_START,
                                            BTN_THUMBL,
                                            BTN_THUMBR]
        </rosparam>
    <rosparam param="axes_mapping">         [ABS_X,
                                            ABS_Y,
                                            ABS_RX,
                                            ABS_RY,
                                            ABS_Z,
                                            ABS_RZ,
                                            ABS_HAT0X,
                                            ABS_HAT0Y]
    </rosparam>
  </node>
</launch>

